Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -2,6 +2,8 @@
 # frozen_string_literal: true
 
 require "abstract_command"
+require "tap"
+require "utils/bottles"
```

Comment:
@chenrui333 Thanks for the PR! Could you add these more tightly scoped i.e. inside the `def run` method?

@apainintheneck Please don't ✅ these `require` PRs that add them globally, thanks ❤️. We want to avoid regressing the performance improvements.

Reply:
This doesn't add them globally though. It is in a command file which will only be loaded when the command gets run. It also fixes a bug which is a higher priority than small performance benefits anyway.

That being said if we don't want there to be top-level requires in command files it would be trivial to create a cop for that. If we can't audit for this sort of thing, it will be very hard to maintain going forward regardless.